src/page/GraphEditorPage.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './GraphEditorPage.css';
+import Graph from '../graph/Graph';
+import Timeline from './Timeline';
+
+class GraphEditorPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            vertices: [],
+            edges: [],
+        };
+
+        // array of XY coords
+        this.vertices = [[0, 1], [2, 3], [-3, 2]];
+        // array of FROM TO vertices
+        this.edges = [[0, 1], [0, 2]];
+    }
+
+    render() {
+        return (
+            // .cockpit-fullscreen contains only one Component
+            // every other component should be placed in .cockpit and given a position: {relative | fixed | absolute | sticky}
```

Comment:
[MINOR] I don't understand the comment `// .cockpit-fullscreen contains only one Component`.
I would also say that this comment belongs the css part not to the logic.